.expeditor/scripts/verify/run_cargo_test.ps1

```diff
@@ -28,6 +28,10 @@ If($Features) {
 
 # Set cargo test invocation
 Install-Rustup $toolchain
+# In this context we are installing a nightly toolchain and do not
+# require any additional components, we'll set a minimal profile.
+# https://blog.rust-lang.org/2019/10/15/Rustup-1.20.0.html#profiles
+rustup set profile minimal
```

Comment:
We already set the profile to minimal in both the bash and powershell `Install-Rustup` and `install_rustup` functions

Reply:
If rustup was already installed then those functions leave the profile as is

Reply:
Great callout @mwrock, thanks!
Handled in https://github.com/habitat-sh/habitat/pull/7274/commits/abf3174a07c0aa6b13d6600db867d53044b4be92

Reply:
we should remove it from the bash script too since we do the same thing there

Reply:
Agreed! https://github.com/habitat-sh/habitat/pull/7274/commits/c45d279a11c63f239b58dbf903139fb2705cc55e takes care of it over there.